src/generators/bendable-amogus/AmongUsBendableGenerator.res

```diff
@@ -0,0 +1,254 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "bendableamogusgenerator"
+
+let name = "Bendable Amogus generator"
```

Comment:
Let's make the name `Among Us Bendable` (no need for the word "generator" here)

Reply:
Done